### ISSUE TITLE
Review aria-label on nav examples

### ISSUE THREAD

**Issue by mcoker** [patternfly/patternfly-next#1779]
Some of our nav examples are missing `aria-label`. Let's review all of the instances of the nav component and ensure the correct `aria-label` is being used.

`aria-label="Global"` should be used for all global (site-wide) navigation.

`aria-label="Local"` should be used for all non-site-wide (ie, tertiary) navigation.

```
src/patternfly//components/Nav/examples/nav-horizontal-list-overflow-example.hbs:    {{#> nav nav--modifier="pf-m-start pf-m-end"}}
src/patternfly//components/Nav/examples/nav-tertiary-list-overflow-example.hbs:{{#> nav nav--modifier="pf-m-start pf-m-end"}}
src/patternfly//components/Nav/examples/nav-simple-list-example.hbs:{{#> nav nav--attribute='id="nav-primary-simple"'}}
src/patternfly//components/Nav/examples/nav-horizontal-list-example.hbs:    {{#> nav}}
src/patternfly//components/Nav/examples/nav-example.hbs:{{#> nav}}
src/patternfly//components/Nav/examples/nav-expandable-example.hbs:{{#> nav}}
src/patternfly//components/Nav/examples/nav-tertiary-list-example.hbs:{{#> nav}}
src/patternfly//components/Nav/examples/nav-expandable-subnav-titles-example.hbs:{{#> nav}}
src/patternfly//components/Nav/examples/nav-list-grouped-example.hbs:{{#> nav}}
src/patternfly//components/Nav/examples/nav-mixed-example.hbs:{{#> nav}}
src/patternfly//demos/Page/examples/page-component-expandable-nav-example.hbs:    {{#> nav nav--expandable="true" nav--attribute=(concat 'id="' page--id '-expandable-nav" aria-label="Global"')}}
src/patternfly//demos/Page/examples/page-component-horizontal-nav-example.hbs:    {{#> nav nav--modifier="pf-m-end" nav--attribute=(concat 'id="' page--id '-horizontal-nav" aria-label="Global"')}}
src/patternfly//demos/Page/examples/page-component-tertiary-nav-example.hbs:    {{#> nav nav--expandable="true" nav--attribute=(concat 'id="' page--id '-tertiary-nav" aria-label="Global"')}}
src/patternfly//demos/Page/examples/page-component-tertiary-nav-example.hbs:      {{#> nav nav--modifier="pf-m-start pf-m-end" aria-label="Local"}}
src/patternfly//demos/Page/examples/page-component-grouped-nav-example.hbs:    {{#> nav nav--attribute=(concat 'id="' page--id '-grouped-nav" aria-label="Global"')}}
src/patternfly//demos/Page/page-demo-default.hbs:    {{#> nav nav--attribute=(concat 'id="' page--id '-primary-nav" aria-label="Global"')}}
src/patternfly//demos/DataList/data-list-main-section-nav.hbs:{{#> nav nav--modifier="pf-m-start pf-m-end" nav--attribute='aria-label="Local"'}}
src/patternfly//demos/DataList/data-list-page-nav.hbs:{{#> nav nav--attribute=(concat 'id="' page--id '-primary-nav" aria-label="Global"')}}
src/patternfly//demos/DataTable/data-table-page-nav.hbs:{{#> nav nav--attribute=(concat 'id="' page--id '-primary-nav" aria-label="Global"')}}
src/patternfly//demos/DataTable/data-table-main-section-nav.hbs:{{#> nav nav--modifier="pf-m-start pf-m-end" nav--attribute='aria-label="Local"'}}
```

**Comment by rachael-phillips** [comment]
Hi @mcoker what priority do you think this is to get done?

**Comment by mcoker** [comment]
It's just a documentation update on the core side, so not a showstopper, but it's an easy fix and if someone were to copy/paste the code from core, they wouldn't get the proper `aria-label` in some cases. So I'd like to get it done ASAP, especially since it's such an easy update. It could go in any time since it doesn't touch the core code - only touches our documentation examples.
